### ISSUE TITLE
[20211205] Weekly AI ArXiv 만담 - 32회차

### ISSUE THREAD

**Issue by jungwoo-ha** [jungwoo-ha/WeeklyArxivTalk#32]
- News
  - AAAI 2021 Notification: 일단 모두들 축하드리고.. 그런데 말도 많고 탈도 많고... (자세한건 정현님이)
  - NeurIPS 2021: 월요일 부터 시작!
    - NAVER CLOVA의 프로그램: https://naver-career.gitbook.io/en/teams/clova-cic/events/clova-ai-lab-neurips-2021
  - [NeurIPS 2021 Social: ML in Korea](https://naver-career.gitbook.io/en/teams/clova-cic/events/neurips-2021-social-ml-in-korea)
    - 12월 6일, 오전 9시 ~ 12시
    - 개더타운 주소는 오전 8시 홈피 통해 공개 (뉴립스 등록자분들만 가능)
    - 한보형 교수님 키노트 + 44개 뉴립스 논문 스팟라잇+포스터 + 8개회사소개/부수 + 선물
  - [NeurIPS WS on ImageNet Past, Present, Future](https://sites.google.com/view/imagenet-workshop/)
    - NAVER AI Lab, NLE, Google Brain, U of Tuebingen 이 공동 주최
    - ImageNet 관련 세계 최고 연구자들이 모여 강연 + 패널 토의
    - 12월 13일 (한국시간 밤 8시 부터 새벽까지 ㅠㅠ)
  - [AI Summit Seoul 2021](https://aisummit.co.kr/)
    - 12월 8일 ~ 9일
  - [LG AI Talk concert](https://www.youtube.com/watch?v=3YyLUbN2Ylg)
    - 12월 14일
  - [사람을 위한 AI: 독거노인분들을 위한 케어콜](https://tv.naver.com/v/23888407)
- ArXiv
  - [BEVT: BERT Pretraining of Video Transformers](https://arxiv.org/abs/2112.01529)
    - [BEiT](https://arxiv.org/abs/2106.08254)의 video version from Fudan Univ + MS Cloud + AI 
    - 기본적으로 VQ-GAN류로 tokenizer 만들고 Masked Image Modeling & Masked video modeling
    - Encoder는 image와 video weight sharing 하고 decoder는 별도
    - 기본 구조는 Swin을 사용함.
    - Pretrain 데이터: ImageNet-1k, K-400 or HowTo100M
    - 다운스트림은 Video recognition (SSv2, Diving-48, K400) --> Zeroshot은 없음.
![image](https://user-images.githubusercontent.com/11782739/144740945-54aef89a-5f02-42ff-bd42-70be0eb41ae4.png)

  - [Benchmarking Detection Transfer Learning with Vision Transformers](https://arxiv.org/abs/2111.11429)
    - 뭔가 MAE의 Detection 실험 파트를 잘라내서 자세하게 리포팅한 것 같은 (from Meta AI: 조직 이름 언제 통일??)
    - Detection은 기본적으로 다른 다운스트림 보다 복잡해서 ViT 나온 이후 공정한 benchmark 만들기 어려웠음
    - 그래서 5가지 ViT 계열에 대해 Mask RCNN 붙이고 Coco 다운스트림 비교 (Self-super, super, random init 까지)
    - 결국 MAE가 젤 좋더라~ (BEiT는 dVAE의 Dall-e 데이터 때문에..)
![image](https://user-images.githubusercontent.com/11782739/144741338-d4267351-837e-486a-b4b1-b8ca9c2ba320.png)

  - [How much do language models copy from their training data? Evaluating linguistic novelty in text generation using RAVEN](https://arxiv.org/abs/2111.09509)
    - 언어모델의 NLG관점에서 정말 새로운 표현을 잘 생성하는 지 아니면 훈련데이터 복사하는지는 중요한 이슈
    - 이것을 모델 별로 (LSTM, Transformer-XL, GPT-2, 사람) 평가하는 지표와 프로토콜인 RAVEN 제안
    - 짧은 단위로는 novelty 가 약함. 문장단위로 넘어가면 novel 한 표현 잘 만들어냄 (전체적으로)
    - 특히 GPT2의 경우 문법적으로는 잘하는데 길어지면 모순되는 내용 생성
    - 자체 초대규모 언어모델 만드는 연구 그룹은 참조하면 좋을 듯
![image](https://user-images.githubusercontent.com/11782739/144741617-bde100b4-85bb-483b-8c99-45c512890849.png)

  - [Uni-Perceiver: Pre-training Unified Architecture for Generic Perception for Zero-shot and Few-shot Tasks](https://arxiv.org/abs/2112.01522v1)
    - 다양한 멀티모달 태스크를 하나의 Transformer encoder를 활용하여 학습하는 구조 (from 센스타임, 홍콩중문대)
    - 딥마인드 Perceiver와는 아무 상관없음 (인용도 안함)
    - input와 target을 동일한 encoder 를 쓰고 joint prob 계산
    - Pretrainig 데이터: 이미지, 텍스트, 비디오 다양한 종류로 많이 씀 (128 V100 실험)
    - Finetuning 보다 1% P-tuning  결과가 인상적 (P-tuning V2 썼다고함)
![image](https://user-images.githubusercontent.com/11782739/144742331-ed44b29b-7511-49e0-9f48-8488981bd750.png)
![image](https://user-images.githubusercontent.com/11782739/144742319-dd4cf22b-6041-468a-8c16-f8ffb5923f3d.png)

**Comment by jnhwkim** [comment]
* [VIOLET : End-to-End Video-Language Transformers with Masked Visual-token Modeling](http://arxiv.org/abs/2111.12681) (ArXiv 2021, Nov)
  * Video pretraining method for 1) text-to-video retrieval (T2VR), 2) zero-shot T2VR, & 3) video QA
  * Pretraining datasets:
    * YT-Temproal-180M using 6M Youtube and its ASR (MERLOT, Zellers et al., 2021)
    * WebVid-2.5M (Bain et al., 2021)
    * CC 3M *image-text* pairs (Sharma et al., 2018)
  * Encoders:
    * Video: dVAE from DALL-E (Ramesh et al., 2021) and Video Swin Transformer (Liu et al., 2021)
    * Language: WordPiece (Wu et al., 2016) and VL-BERT (Su et al., 2020)
  * Losses:
    * Masked Visual-token Modeling (MVM) exploiting the dVAE indices for visual tokens.
    * Visual-Text Matching (VTM) and Masked Language Modeling (MLM), which are noting new ones
    * Blockwise (Bao et al., 2021; Tan et al., 2021) and attended (Zellers et al., 2021) maskings
  * **Overall assessment**:
    * A shallow contribution with the MVM with dVAE but with many upgraded parts
 
<img width="508" alt="image" src="https://user-images.githubusercontent.com/1910051/143971888-16504933-f433-4aa4-ba93-ca72a298f0bb.png">
<img width="1056" alt="image" src="https://user-images.githubusercontent.com/1910051/143971921-34792657-8495-49e9-a82f-e12aa2397496.png">

**Comment by nick-jhlee** [comment]
~~Downfall of AAAI 2022...?~~ AAAI 2022 결과
- AAAI 2022 결과가 나왔는데,,, 네 일단 나왔어요 (accept되신 분들 모두 축하드립니다! reject되신 분들은 너무 상심하지 마세요,, see below)
- Good news: Naver CLOVA에서 4개, 저 1개 <- AI로 저의 *데뷔* 논문입니당! Classical(?) ML논문으로 써보았습니다,,
  -  Meta-Learning for Online Update of Recommender Systems
  - COVID-EENet: Predicting Fine-Grained Impact of COVID-19 on Local Economies
  - ALP: Data Augmentation using Lexicalized PCFGs for Few-Shot Text Classification
  - [BROS: A Pre-trained Language Model Focusing on Text and Layout for Better Key Information Extraction from Documents](https://arxiv.org/abs/2108.04539?fbclid=IwAR1Q1bUA7en0tomK3K8PoGTrZlUUJwE7h8TfcnWF0F3k7ZjzpLHteUNF7w)
  - [Fast and Efficient MMD-based Fair PCA via Optimization over Stiefel Manifold](https://arxiv.org/abs/2109.11196)
  
- Bad news: 리뷰 process가... 최선을 다했다면 할 말이 없지만 뭔가... 흠
  - Review가 늦었는데 rebuttal deadline은 유지..
  - final decision이 예정보다 이틀 늦었는데, 아무런 official announcement 없었음...
    - 나중에 어떤 분이 메일을 보내니까 "The decisions will go out tomorrow. Sorry for the delay"라는 답변을 받고, 그걸 twitter에 올려서 겨우 알게됨..
  - "This year we received a record **9,251 submissions**, of which 9,020 were reviewed. Based on a thorough and rigorous review process we have **accepted 1,349 papers**. This yields an overall acceptance rate of **15%**."
    - 이렇게 적게 하면 randomness가 커지는건 trivial한데,,, 단순 행정적인 이유..?
  - 많은 일들이 있었습니다...
    - first phase에서 억울하게 탈락한 논문들 (e.g. reviewer가 assign된 paper를 잘못알아서 탈락된 경우..)이 있었는데, PC(+AC?)는 안돼 돌아가시전..
    - review의 quality에도 의문이 약간 드는 부분.. (ex. theoretical results were "uninteresting to mainstream readers")
      - 이 분은 트위터에 "I don't submit to @RealAAAI anymore."라고 손절하셨습니다 ㅎㅎ
    - AC(PC?)가 어떤 reviewer의 review를 마음대로 건드린 일도 있었음
    - 4 accept였던 paper가 최종 reject 먹은 일도 있었음
  - 전체적으로 행정적으로 마음에 안들었음,,
  - 실제 행사에서도 이러면 아주 실망... (~~1.5티어 확정...?~~)
  
  
Upcoming conference deadlines:
  - IJCAI 2022
  - ICML 2022
  - COLT 2022
  - ...etc.
  
Upcoming conference attendances:
  - NeurIPS 2022 (online)
  - AAAI 2022 (Vancouver, BC, Canada)
  - ICSE 2022 (Pittsburgh, PA, USA)
  - ...etc.

**Comment by nick-jhlee** [comment]
<Papers>

- [Hashing it Out: A Survey of Programmers’ Cannabis Usage, Perception, and Motivation](https://github.com/CelloCorgi/HashingItOut_ICSE2022) (ICSE 2022)
  - University of Michigan
  - cannabis=대마초

- [Fast and Efficient MMD-based Fair PCA via Optimization over Stiefel Manifold](https://arxiv.org/abs/2109.11196)
  - KAIST(저 포함), UC Berkeley(->Citadel), UIUC
  -

**Comment by kimyoungdo0122** [comment]
- News
    - MODUCON 21 성황리에 마무리 됐습니다!
        - [하정우 소장님의 18년도 모두콘 키노트 발표](https://youtu.be/PDKepFANXT8)
        - [진원님의 19년도 모두콘의 on-device AI 모델 경량화 발표](https://youtu.be/F5yP3kRLM8E)
        - 내년에도 기대해주세요~!
    - 팀닛 게브루가 직접 설립한 AI 윤리 연구소 개설
        - [Google fired its star AI researcher one year ago. Now she’s launching her own institute](https://www.washingtonpost.com/technology/2021/12/02/timnit-gebru-dair/)
        - [DAIR(Distributed AI Research)](https://www.dair-institute.org/)
        - 거대 자본에 영향을 받지 않는 독립적인 AI 윤리 연구를 하겠다
        - [ML에 사용할 dataset에 대한 Datasheets for Datasets](https://cacm.acm.org/magazines/2021/12/256932-datasheets-for-datasets/fulltext)
        - dataset 구축 시 고려했던 요소들, 윤리적으로 문제의 소지가 있는 데이터를 어떻게 걸러냈는지 등등을 data description을 풍부하게 작성하자
        - [U.S. prisons are installing AI-powered surveillance to fight crime](https://news.trust.org/item/20211115095808-kq7gx/)
    - 네이버 랩스의 디지털 트윈
        -

**Comment by veritas9872** [comment]
이번 주는 PyTorch Opacus 라이브러리 1.0 stable version이 나오면서 Privacy preserving AI에 대해 소개해드리고자 합니다.
White paper가 나온지 조~금 되었지만 아직까지 Arxiv talk에서 differential privacy에 대한 논의가 소개된 적이 없어 공유합니다.

![image](https://user-images.githubusercontent.com/33523965/144746784-bb116229-b049-4290-bde8-ed5032f1dcaf.png)

GitHub: https://github.com/pytorch/opacus
Website: https://opacus.ai
Arxiv: https://arxiv.org/pdf/2109.12298.pdf
Blogposts:
1. https://medium.com/pytorch/differential-privacy-series-part-1-dp-sgd-algorithm-explained-12512c3959a3
2. https://medium.com/pytorch/differential-privacy-series-part-2-efficient-per-sample-gradient-computation-in-opacus-5bf4031d9e22

Differential privacy란 딥러닝에서 발생하는 training data memorization을 해결하고자 하는 연구 분야입니다.
예를 들어 우리나라에서 크게 이슈가 되었던 "이루다"는 질문에 대해서 학습 데이터에 포함되어 있던 특정 사용자의 집 주소와 같은 민감 정보를 공개하는 문제를 보여서 서비스가 종료되었습니다. Differential privacy는 학습에서 개별 데이터의 영향력을 특정 범위 내로 한정함으로써 이러한 문제를 완화하고자 합니다.

![Screenshot (75)](https://user-images.githubusercontent.com/33523965/144747092-9aee0c37-fec9-46ae-813b-cd809eb182b4.png)

오늘 소개드리는 Opacus는 페이스북(메타라고 쓰는걸 잊어버렸네요 ㅋㅋㅋ)에서 공식적으로 PyTorch에 호환되도록 만든 라이브러리로 위에 보시는 것처럼 기존 모델에 매우 단순하게 호환할 수 있습니다.

Opacus에서 Differential privacy를 구현하는 방식은 각 mini-batch에서 sample마다 gradient를 clipping하거나 regularization함으로써 개별 sample의 영향력을 한정시키고자 하는데 JAX를 제외한 다른 라이브러리에서는 평균 gradient만 접속할 수 있지 sample-wise gradient를 접속하는 방법은 구현되어있지 않습니다.

이를 해결하기 위해 einsum을 사용해 sample-wise gradient를 계산하는 방식을 사용하고 differential privacy가 한정치를 넘어가는 시점에서 early stopping을 하거나 개별 gradient에 noise를 주는 방식 등을 적용해서 연산을 진행한다고 합니다.

최근 인공지능이 privacy에 어떤 영향을 미칠지에 대해 많은 논의가 진행되었는데 연구가 필요한 분야라고 생각됩니다.

**Comment by hollobit** [comment]
### AI to see stricter regulatory scrutiny starting in 2022, predicts Deloitte
https://www.techrepublic.com/article/ai-to-see-stricter-regulatory-scrutiny-starting-in-2022-predicts-deloitte/
- 인공 지능 규제에 대한 논의가 내년에 본격화될 것이며, 2023년에는 실제 규칙이 뒤따를 것이라고 딜로이트가 예측
- 지금까지 비즈니스 분야에서 새로운 기술 중심 접근으로 규제 회피
- 프라이버시 및 기타 민감한 영역에 대한 우려가 높아짐에 따라 규제 가능성 높아짐
-  올해 유럽 연합과 미국 연방 거래 위원회(FTC)는 AI를 보다 엄격하게 규제하기 위한 제안과 문서를 작성했습니다. 중국은 기술 회사를 규제하는 일련의 규정을 제안했으며 그 중 일부는 AI 규정을 포함
- 규제 기관이 AI를 더 면밀히 주시하는 데에는 몇 가지 이유
 1) 첫째, 이 기술은 몇 년 전보다 훨씬 강력하고 유능
 2) 둘째, 규제 기관은 기계 학습 사용에 거의 고유한 사회적 편견, 차별 및 개인 정보 보호 문제에 대해 더 많이 우려
 3) 규제 진입장벽.  어느 한 국가나 정부가 자체 AI 규정을 설정하면 해당 지역 기업이 다른 국가의 기업보다 이점을 얻음 

### 높아지는 AI 책임성에 대한 사례들 
https://www.wired.com/story/movement-hold-ai-accountable-gains-steam/
- 뉴욕시 법에 따르면 채용에 사용되는 알고리즘은 편견에 대해 "감사"해야 함. 이는 미국 최초 규제
- 뉴욕 시의회는 지난달 고용이나 승진에 고용주가 사용하는 알고리즘에 대한 감사를 요구 하는 법률을 채택
- 유럽 연합 의원들은 고위험으로 간주되는 AI의 검사를 요구 하고 고위험 시스템의 공개 레지스트리를 만드는 법안을 고려
- 일부는 회사 재무 감사와 유사한 알고리즘의 의무 감사 방식을 선호, 그러나 한편에서는 환경 영향 보고서와 유사한 "영향 평가" 방식을 선호. 
- 검토가 어떻게 수행되어야 하고 무엇을 포함해야 하는지에 대한 표준이 현장에서 절실히 필요하다는 데 동의
- 표준이 없으면 기업은 유리한 감사를 주선하여 "윤리 세척"에 참여

Why international cooperation matters in the development of artificial intelligence strategies
https://www.brookings.edu/blog/techtank/2021/11/30/why-international-cooperation-matters-in-the-development-of-artificial-intelligence-strategies/
https://www.brookings.edu/research/strengthening-international-cooperation-on-ai/
- 브루킹스 연구소와 유럽정책연구센터가 2020년 설립한 Forum for Cooperation on Artificial Intelligence (FCAI)에서 인공지능 국제 협력에 관한 중간 보고서 발표 
- AI에 대한 국제 협력이 중요한 이유 7가지 이유를 비롯해 AI에 대한 국제 협력을 더욱 발전시키기 위한 15가지 구체적인 권장 사항을 제시
1) AI 연구 및 개발은 규모가 중요한 이점인 점점 더 복잡하고 자원 집약적인 노력
2) 책임 있는 AI에 대해 일반적으로 합의된 민주주의 원칙에 기반한 국제 협력은 책임 있는 AI 개발에 집중하고 신뢰를 구축하는 데 도움이 될 수 있음
3) 규제와 관련하여 다양한 접근 방식은 혁신과 확산에 장벽을 만들 수 있음
4) AI 규제의 주요 측면을 조정하면 AI 개발 전문 기업이 번창할 수 있음
5) 무역 협력 강화는 상품과 데이터의 흐름에 대한 부당한 제한을 피하기 위해 필수적이며, 이로 인해 AI 확산의 잠재적 이점이 크게 감소
6) AI 솔루션의 잠재력을 활용하여 글로벌 문제를 해결하려면 협력을 강화해야 함
7) 민주주의, 표현의 자유 및 기타 인권의 개방과 보호의 핵심 원칙을 재확인하기 위해서는 같은 생각을 가진 국가 간의 협력이 중요

**Comment by nick-jhlee** [comment]
- 지나간 행사: 제2회 인공지능 심포지엄 @ 코엑스
  - 인공지능 대학원 총 집합 + 관련 기업분들 부스
  - NAVER, NVIDIA talk (research collaboration)
  - 패널 토크..등등
  - booths
     - NAVER souvenier 감사드립니다 ㅎㅎ
  - 학생분들이 없어서 약간 아쉬웠습니다 ㅠㅠ
  - 그래도 뭔가 같이 교류하는 장이 있어서 좋았습니다..! (좀 더 많이 활성화되면 정말 좋을 듯 하네용!)
